test/functional/test_framework/util.py

```diff
@@ -183,7 +183,7 @@ def check_json_precision():
 def count_bytes(hex_string):
     return len(bytearray.fromhex(hex_string))
 
-def hash256(byte_str):
+def hash256r(byte_str):
```

Comment:
wouldn't mind being extra verbose here, since it is only used twice in the code

```suggestion
def hash256_reversed(byte_str):
```

Reply:
Or maybe just move it to the zmq test, as that is the only place where it is used